src/popup/App/OpenExtensionInBrowserIndicator/OpenExtensionInBrowserIndicator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import OpenInBrowserIndicator from 'src/popup/components/OpenInBrowserIndicator';
+
+function OpenExtensionInBrowserIndicator(props) {
+
+  return (
+      <OpenInBrowserIndicator
+          icon={
+              <svg width="16" height="16" viewBox="0 0 433 466" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think it'd be better to have all icons in a single place. Today it's in the `IconButton` component https://github.com/filecoin-shipyard/browser-retrieval/blob/master/src/popup/components/IconButton/IconButton.js

By doing this you could potentially use a single component too (e.g. keep the other and replace button/icon with `IconButton`)

Reply:
@ionutiorgulescu Can you see if you can get this to work in at least Chrome?  Edge would be ideal too.

Also, I agree about having all the icons in one place.

Reply:
Just tested, it works in Chrome and Edge too. I will move the icon to IconButton component then give back to code review. Thanks guys!